plugins/rebalance/rest.go

```diff
@@ -0,0 +1,231 @@
+package rebalance
+
+import (
+	//"context"
+	//"encoding/json"
+	"fmt"
+
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
```

Comment:
A comment on style. We group packages into blocks based on where they're imported from. The blocks are in the following order,
```go
import (
  // Standard library packages first
  "os"
  "fmt"

  // Packages from GD2 project in the middle
  "github.com/gluster/glusterd2/glusterd2/gdctx"

  // External packages last
  log "github.com/sirupsen/logrus"
)
```

Each block is also sorted alphabetically. To make this easier you should set up you editor with plugins that run 'goimports` when you save your file.